actions.yaml

```diff
@@ -0,0 +1,6 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+rotate-secret-key:
+  description: >
+    rotating the flask secret key. Users will be forced to log in again.
+    This might be useful under security breach circumstances.
```

Comment:
s/security breach circumstances/is a security breach occurs? I'm thinking about keeping it as simple as possible for documentation purposes

Reply:
Updated, thanks!